RecoTracker/SpecialSeedGenerators/python/cosmicDC_cff.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+# seeding
+from RecoMuon.CosmicMuonProducer.cosmicMuons_cfi import *
+import RecoMuon.MuonIdentification.muons1stStep_cfi
+muonsForCosmicDC = RecoMuon.MuonIdentification.muons1stStep_cfi.muons1stStep.clone(
+    inputCollectionLabels = cms.VInputTag("cosmicMuons"),
+    inputCollectionTypes = cms.vstring('outer tracks'),
+    fillIsolation = cms.bool(False),
```

Comment:
Using muonsFromCosmic, gives the following error:
----- Begin Fatal Exception 20-May-2016 11:06:12 CEST-----------------------
An exception of category 'MuonDetLayerMeasurements' occurred while
   [0] Processing run: 1 lumi: 1 event: 6
   [1] Running path 'reconstruction_step'
   [2] Calling event method for module CosmicMuonSeedGenerator/'CosmicMuonSeed'
Exception Message:
Cannot get DT RecHits
----- End Fatal Exception -------------------------------------------------